portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,29 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/random-fact")
 public class DataServlet extends HttpServlet {
 
+  static final List<String> facts = new ArrayList<String>() {{
```

Comment:
Is there a fixed-size constructor? You may also be able to make it an immutable list.

Reply:
There is an immutable list object provided by google to open source. To use it requires adding a dependency to the project to use it.

I'm not sure how to add a dependency but I'm doing some research on how to do that with the maven set up we have on cloud shell.